controller.py

```diff
@@ -211,58 +210,7 @@ def canonicalize_author_name(self):
         return make_response(author_name, HTTP_OK, {"Content-Type": "text/plain"})
 
 
-class ISBNEntryMixin(object):
```

Comment:
This whole thing is gone because ISBNs will reliably get works now. (Though we may need some kind of migration to create Works for ISBNs that have already been processed but didn't get Works.)